fcrepo-http-commons/src/main/resources/context.jsonld

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In older versions of fedora, the context appears to be dynamically built to include every property as its own context entry, including properties from outside of these standard ones. Amusingly, compact is longer than expanded when I try it in my fedora 5 repo.

I'm not really sure we need to reproduce the previous behavior, and maybe this is a good compromise. Another option might be to use the repositories list of [RDF namespaces from registry](https://wiki.lyrasis.org/display/FEDORA6x/RDF+Namespaces), but that would be an even bigger change. So I'm not really asking for a change here, just raising how this differs.